web/v1/bigquery/sql-runner/sql/custom/02-custom-example/01-link-clicks.sql

```diff
@@ -0,0 +1,38 @@
+-- We use entropy templates to allow ourselves easier separation of dev and prod runs.
+CREATE OR REPLACE TABLE {{.scratch_schema}}.link_clicks{{.entropy}} AS(
+  WITH click_count AS(
+    SELECT
+      ev.page_view_id,
+      COUNT(DISTINCT ev.event_id) AS link_clicks,
```

Comment:
Just a question: Here it seems all events are counted per `page_view_id`, not only the `link_click`s. Is this the case?

Reply:
Oh true. Good spot, thanks for flagging. Have pushed a fix!